pkg/api/devops/v1alpha3/pipeline_types_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2020 The KubeSphere Authors.
```

Comment:
Now is `2022` instead of `2020`

Reply:
Thanks for your patient review. This file was auto-generated by VSCode, and I will correct it later.